predicators/approaches/active_sampler_learning_approach.py

```diff
@@ -54,6 +56,9 @@ def __init__(self, initial_predicates: Set[Predicate],
         # successfully reached their effects or not). Updated in-place by the
         # explorer when CFG.explorer is active_sampler_explorer.
         self._ground_op_hist: Dict[_GroundSTRIPSOperator, List[bool]] = {}
+        self._ground_op_competence_data: Dict[_GroundSTRIPSOperator,
+                                              Tuple[List[float],
+                                                    List[float]]] = {}
```

Comment:
can you add a comment explaining what the values are (i.e, what do the two lists here represent)?